src/ui/pages/AssetInfo/AssetResources.tsx

```diff
@@ -0,0 +1,365 @@
+      ) : (
+        <CopyIcon className="content-hover" />
+      )}
+    </UnstyledButton>
+  );
+}
+
+function AssetImplementationsDialog({
+  implementations,
+}: {
+  implementations: { address: string | null; network: NetworkConfig }[];
+}) {
+  const { networks } = useNetworks();
+  return (
+    <VStack
+      gap={14}
+      style={{
+        ['--surface-background-color' as string]: 'var(--white)',
+        padding: '8px 16px 24px',
+      }}
+    >
+      <VStack gap={0}>
+        {implementations.map(({ address, network }) => (
+          <HStack
+            key={network.id}
+            gap={24}
+            justifyContent="space-between"
+            alignItems="center"
+            style={{ paddingBlock: 12 }}
+          >
+            <HStack gap={12} alignItems="center">
+              <img
+                src={network.icon_url}
+                alt={network.name}
+                width={36}
+                height={36}
+              />
+              <VStack gap={0}>
+                <UIText kind="body/accent">{network.name}</UIText>
+                {address ? (
+                  <UIText kind="small/regular" color="var(--neutral-500)">
+                    {middleTruncate({ value: address })}
+                  </UIText>
+                ) : null}
+              </VStack>
+            </HStack>
+            <HStack gap={0}>
+              {address ? <CopyAddressButton address={address} /> : null}
+              <UnstyledAnchor
+                rel="noopener noreferrer"
+                target="_blank"
+                href={
+                  address && networks
+                    ? networks.getExplorerTokenUrlByName(
+                        createChain(network.id),
+                        address
+                      )
+                    : network.explorer_home_url || ''
+                }
+                className="parent-hover"
+                style={{
+                  ['--parent-content-color' as string]: 'var(--neutral-400)',
+                  ['--parent-hovered-content-color' as string]:
+                    'var(--neutral-700)',
+                  display: 'flex',
+                }}
+              >
+                <LinkIcon className="content-hover" />
+              </UnstyledAnchor>
+            </HStack>
+          </HStack>
+        ))}
+      </VStack>
+    </VStack>
+  );
+}
+
+function ResourseButton({
+  resource,
+  icon,
+}: {
+  resource: AssetResource;
+  icon: React.ReactNode;
+}) {
+  return (
+    <Button
+      key={resource.name}
+      as={UnstyledAnchor}
+      rel="noopenner noreferrer"
+      target="_blank"
+      kind="regular"
+      size={36}
+      href={resource.url}
+      style={{
+        padding: 6,
+        border: '2px solid var(--neutral-200)',
+        ['--button-background' as string]: 'var(--white)',
+        ['--button-background-hover' as string]: 'var(--neutral-200)',
+      }}
+      aria-label={resource.displayableName}
```

Comment:
I think `title` is still better here, because I didn't understand what the icon was :)